src/EosForm.tsx

```diff
@@ -24,23 +22,26 @@ const EosForm = ({
 
   return (
     <div>
-      <label htmlFor="searchInput" className="searchInput">
-        <input
-          ref={inputField}
-          type="text"
-          id="searchInput"
-          placeholder="Search..."
-          onKeyUp={handleKeyUp}
-        />
-        <div onClick={clearValue}>
-          <EOS_CLOSE_OUTLINED size="sm" />
+      <div className="searchBar">
+        <div style={{ margin: "auto 5 auto 5" }}>
+          <img
+            src="https://img.icons8.com/glyph-neue/24/000000/search--v1.png"
+            alt=""
+            style={{ width: 20, height: 20 }}
+          />
         </div>
-      </label>
+        <div className="inputField">
```

Comment:
`input-field` since we are following this throughout the codebase here

Reply:
yeah, I had read EOS wiki for conventions for CSS, may be by mistake I have wrote that.